### PR TITLE
v17 Upgrade Working Branch

### PR DESCRIPTION
**Pull request by Reecepbcups**
## Description
- Renamed from v16 -> v17 + Added v17 base upgrade handler 6ce4004a337bcb7efa734d1a88a583174cd6bed4

### CODE REVIEW

**Comment by Reecepbcups** [comment]
v17 will be wasm-client, DAODAO register contracts natively, and maybe x/drip

**Review by JakeHartnell — APPROVED** [review]
`v16` -> `v17` ✅ 

LFG!

**Comment by elix1er** [comment]
Sum eli5 of wasm-client, DAODAO register contracts natively, and maybe x/drip plz

Happening ?

**Comment by Reecepbcups** [comment]
@elix1er 
> Sum eli5 of wasm-client, DAODAO register contracts natively, and maybe x/drip plz

wasm-client - connect to polkadot & kusama natively over IBC

DAODAO Register contracts - DAODAO can register admin of contracts natively even if they are a factory contract

x/drip - allow tokens to be sent to all token holders in a single block without any gas cost, easier for a massive airdrop
